user-guide/Advanced_Modules/User Defined APIs/Defining_an_API/UD_APIs_Define_New_API.md

```diff
@@ -151,25 +151,30 @@ You can also use the `StatusCode` enum, which contains suggestions, and cast tha
 > - A simple example is included in the [Automation script example](#creating-the-api-automation-script).
 > - For more insight into which HTTP status codes to use in which circumstances, see [HTTP status codes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).
 
-## Creating an API definition and token(s)
+## Creating an API and token(s) in Automation
 
-1. Open your API script in the Automation app in DataMiner Cube and click *Configure API* at the bottom of the window.
+1. Open your API script in the Automation app in DataMiner Cube and click *Configure API...* at the bottom of the window.
```

Comment:
To match the [Microsoft style](https://learn.microsoft.com/en-us/style-guide/punctuation/ellipses), we do not usually add the ellipsis when referring to a UI item with ellipsis.

![image](https://user-images.githubusercontent.com/94605575/232770898-5f73bcbe-6780-45c0-9db3-7afa92b10dc1.png)


```suggestion
1. Open your API script in the Automation module in DataMiner Cube and click *Configure API* at the bottom of the window.
```